spec/models/manageiq/providers/google/cloud_manager/provision_workflow_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe ManageIQ::Providers::Google::CloudManager::ProvisionWorkflow do
+  include Spec::Support::WorkflowHelper
+
+  let(:admin) { FactoryGirl.create(:user_with_group) }
+  let(:ems) { FactoryGirl.create(:ems_google) }
+  let(:template) { FactoryGirl.create(:template_google, :name => "template", :ext_management_system => ems) }
+  let(:workflow) do
+    stub_dialog
+    allow_any_instance_of(User).to receive(:get_timezone).and_return(Time.zone)
```

Comment:
@d-m-u - You should be able to swap out line 9 with `allow(User).to receive_messages(:server_timezone => "UTC")` to get rid of one of the Rubocop warnings about `allow_any_instance_of`